apps/greenhouse/apps/utils/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@cloudoperators/juno-app-greenhouse-utils",
```

Comment:
we should add at least private true so it does not get accidentally published

Reply:
👍